package.json

```diff
@@ -178,7 +178,7 @@
     "cross-env": "^7.0.3",
     "css-loader": "^6.2.0",
     "cssnano": "^6.0.0",
-    "cypress": "^12.14.0",
+    "cypress": "^13.2.0",
```

Comment:
I think Cypress v13 is causing tests to fail.

But only on CircleCI. Everything looks good when I run them locally. So I'm not sure what's going on.

On the test replays it looks like the CSS isn't being loaded properly:

![Screenshot 2023-09-15 at 2 29 04 PM](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/cad57327-6b96-4b8d-9dc2-1ad296c47b9f)

Reply:
So what's cool about v13 is that we should be able to inspect the source maybe on cloud.cypress.io.. I'm not entirely sure though how in depth it gets or if it is still just essentially screenshots..